core/lib/multivm/src/tracers/validator/mod.rs

```diff
@@ -133,7 +139,11 @@ impl<H> ValidationTracer<H> {
         // The user is allowed to touch its own slots or slots semantically related to him.
         let valid_users_slot = address == self.user_address
             || u256_to_account_address(&key) == self.user_address
```

Comment:
Interesting, but not an issue. `u256_to_account_address` just takes the last 20 bytes, and doesn't enforce that the leading 12 bytes are zeroes. The original EIP proposed to not allow key with address + dirty leading bytes.